controls/richtexteditor/overview.md

```diff
@@ -10,11 +10,11 @@ CTAControlName: RichTextEditor
 previous_url: richtexteditor, richtexteditor-troubleshooting, richtexteditor-styles-and-templates-template-structure, richtexteditor-styles-and-templates-styling-the-richtexteditor, richtexteditor-styles-and-templates   
 ---
 
```

Comment:
This article can be more informative. We could add the following, for example:
* A paragraph with a link to the Getting Started for the specific control.
* Next Steps section is also an option.
* The best features of the specific control
* How it works (high-level)
* The common scenarios to use this control